src/main/java/database/ParkingDataBase.java

```diff
@@ -0,0 +1,24 @@
+package database;
+
+import java.sql.*;
+import java.util.*;
+
+import parking.*;
+
+public class ParkingDataBase {
+  public static void addParkingSpot(ParkingSpot p) throws SQLException {
+  }
+
+  public static void removeParkingSpot(ParkingSpot p) throws SQLException {
+  }
+
+  public static void rentParkingSpot(ParkingSpot p, int buyerID) throws SQLException {
+  }
+
+  public static void unRentParkingSpot(ParkingSpot p, int buyerID) throws SQLException {
+  }
+
+  public static List<ParkingSpot> searchParkingSpots(String date, int locX, int locY, String startTime, String endTime) throws SQLException {
```

Comment:
```suggestion
  public static List<ParkingSpot> searchParkingSpots(String date, String locX, String locY, String startTime, String endTime) throws SQLException {
```
Coordinates can't be int. They are at most double, and probably should be saved as strings.